src/scripts/main.js

```diff
@@ -2,7 +2,40 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
```

Comment:
The given task requires the creation of the table headers. Add a function to create the table headers with the specified column names: 'name', 'gender', 'born', 'died', 'age', 'century'.